### PR TITLE
Nexus 4611 it

### PR DESCRIPTION
**Pull request by velo**
Current logs saying plugin is trying to connect to localhost:80... this port is no available on linux.

Made sure port is over 1024

### CODE REVIEW

**Comment by adreghiciu** [comment]
+1

**Comment by peterlynch** [comment]
+1 after log message and ProxyConfig constructor use same values.

File: nexus-maven-plugins/nexus-maven-plugin/src/main/java/org/sonatype/nexus/plugin/AbstractStagingMojo.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ protected final synchronized StageClient connect()
         ProxyConfig proxyConfig = null;
         if ( getProxyHost() != null )
         {
+            getLog().info( "Proxying nexus through: " + getProxyHost() + ":" + getProxyPort() );
             proxyConfig = new ProxyConfig( getProxyHost(), getProxyPort(), getProxyUsername(), getProxyPassword() );
```
Comment (peterlynch) [inline nexus-maven-plugins/nexus-maven-plugin/src/main/java/org/sonatype/nexus/plugin/AbstractStagingMojo.java]: Perhaps call getProxyHost and getProxyPort once only - else you risk the logged message masking a different version that is actually used. Potential future bug should someone change what getProxyPort and getProxyHost implementations.

Comment (velo) [inline nexus-maven-plugins/nexus-maven-plugin/src/main/java/org/sonatype/nexus/plugin/AbstractStagingMojo.java]: done

**Pull request merged**